torch/csrc/jit/codegen/cuda/ops/alias.cpp

```diff
@@ -140,6 +141,44 @@ TensorView* flatten(TensorView* x, int64_t start_dim, int64_t end_dim) {
   return out;
 }
 
+TensorView* squeeze(TensorView* x, const std::vector<bool>& to_squeeze) {
```

Comment:
> Do you like the bool vector for squeeze over list of ints? Seems this is why you put a TODO to replace the trivial reductions in view with squeeze? Or were you just worried about blanket enabling squeeze at this point?

`BroadcastOp` uses a bool vector, and `SqueezeOp` is symmetric with `BroadcastOp`, so I made them similar. The reason why I put a TODO to reduction, is mainly because I want first to find all places that should be directly using squeeze but is using reduction. After that, I will start to change the reduction to do squeeze.